code/modules/assembly/assembly.dm

```diff
@@ -29,10 +30,14 @@
 	var/next_activate = 0 //When we're next allowed to activate - for spam control
 	var/activate_delay = 30
 
+/obj/item/assembly/Initialize()
+	. = ..()
+	secured &&= securable
```

Comment:
I'm not sure if it's right use of this operator

Reply:
The goal of this line is to ensure the assembly starts unsecured if it is not securable. This can be done with short circuiting with the expression secured = securable && secured. The commutative property of logical and allows for the swapping of the order of the operands, which can then be condensed into the statement used with the logical and assignment operator. I can change it back if you want for readability, but it’ll be a while until I’m at my computer to do that.

Reply:
Why not just do secured = securable?

Reply:
That could work, but I’m only concerned about unsecurable assemblies being secured, which could break them. With &&=, secured is only affected when securable is false. Maybe someone wants to have a securable assembly initially unsecured in the future, and this line, even if only 2 characters different, supports that.